data/procs/apps.yml

```diff
@@ -0,0 +1,16 @@
+apps:
```

Comment:
maybe not in this PR, but somewhere it would be nice to have some documentation around this and what it would take for it to be generated at build time

Reply:
Yeah, I think this can go with my upcoming Makefile changes, since we'd probably want to generate it around the time everything gets built.